drive/client.bal

```diff
@@ -83,7 +83,7 @@ public client class Client {
     # 
     # + filterString - The filter string
```

Comment:
Shall we put google doc how this string should come?

Reply:
Updated with link to readme